packages/@aws-cdk/aws-servicecatalog/lib/portfolio.ts

```diff
@@ -161,6 +169,10 @@ abstract class PortfolioBase extends cdk.Resource implements IPortfolio {
     AssociationManager.notifyOnStackEvents(this, product, topic, options);
   }
 
+  public constrainProvisioningParameters(product: IProduct, assertion: TemplateRule, options: CommonConstraintOptions = {}): void {
```

Comment:
I don't love this method name...

How about `constrainCloudFormationParameters()`?